featuretools/primitives/standard/transform/binary/subtract_numeric.py

```diff
@@ -32,6 +32,7 @@ class SubtractNumeric(TransformPrimitive):
     return_type = ColumnSchema(semantic_tags={"numeric"})
     compatibility = [Library.PANDAS, Library.DASK]
     description_template = "the result of {} minus {}"
+    commutative = True
```

Comment:
This may be out of scope, but my initial thought when reading `commutative=True` is that the primitive is commutative

Reply:
Subtraction is not commutative though. Shouldn't this be `False`?

Reply:
yeah, I agree, but it was set as true in the constructor, so just moved that out. Setting it to false would change a lot of tests potentially. 

I think its not a huge deal to leave it as true cause the difference would just be the sign, which shouldn't affect Machine learning

Reply:
Agree on the impact on ML - probably no need to generate 2 perfectly correlated features. Maybe we should make a note of that logic here for future reference?

Reply:
MR opened here: https://github.com/alteryx/featuretools/pull/2527